networkx/algorithms/cluster.py

```diff
@@ -49,6 +49,12 @@ def triangles(G, nodes=None):
     >>> print(list(nx.triangles(G, [0, 1]).values()))
     [6, 6]
 
+    The total number of unique triangles in `G` can be determined by the total
+    triangles-per-node divided by 3, the number of nodes per triangle:
```

Comment:
```suggestion
    The total number of unique triangles in `G` can be determined by summing
    the number of triangles for each node and dividing by 3 (because a given
    triangle gets counted three times, once for each of its nodes).
```